typescript/src/bitcoin.ts

```diff
@@ -603,3 +603,12 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
   const locktimeBE: Buffer = Hex.from(locktimeLE).reverse().toBuffer()
   return BigNumber.from(locktimeBE).toNumber()
 }
+
+/**
+ * Creates the output script from the BTC address.
+ * @param address BTC address.
+ * @returns The un-prefixed and not prepended with length otput script.
+ */
+export function createOutputScriptFromAddress(address: string): string {
+  return Script.fromAddress(address).toRaw().toString("hex")
+}
```

Comment:
Can we cover this function in `bitcoin.test.ts`? It seems `bcoin` [will cover](https://github.com/bcoin-org/bcoin/blob/b0058696cc10c8f9b17190b31fd2cd907d85d047/lib/script/script.js#L1621-L1637) all types of addresses [allowed](https://github.com/keep-network/tbtc-v2/blob/2254cf67e2b3526789f8b34c2a7d66277fb42a3c/solidity/contracts/bridge/Bridge.sol#L444-L446) by the bridge for redemption, but we should use this function as a black box and have it covered with tests to ensure this keeps working as advertised no matter the implementation.

Reply:
07693eb